announcements/2020-Week-42/announcement.md

```diff
@@ -0,0 +1,14 @@
+# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [NGINX Ingress Controller app v1.10.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1100---2020-10-07) upgrades ingress-nginx-controller from `v0.35.0` to `v0.40.2` improving performance and reliability.
+2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. SEs are working with customers to prepare.)
```

Comment:
@ced0ps any idea on the value this upgrade brings?

Reply:
First major release of `kong-ingress-controller` with:
- https://github.com/kong/kubernetes-ingress-controller/compare/0.10.0...1.0.0
- https://github.com/kong/kubernetes-ingress-controller/compare/0.9.1...0.10.0

Fixes of the chart:
- Removed the status field from the TCPIngress CRD.
- Changes to annotation block generation prevent incorrect YAML indentation when specifying annotations via command line arguments to Helm commands. (#200)

Improvements:
- The chart now adds namespace information to manifests explicitly. This simplifies workflows that use helm template.